_config.yml

```diff
@@ -29,9 +29,7 @@ comments:
     num_posts            : # 5 (default)
     colorscheme          : # "light" (default), "dark"
 feed:
```

Comment:
Testing this I hit an error. Maybe change `feed` to `atom_feed` since I believe the previous name was conflicting with the `jekyll-feed` plugin.

Also if you could squash all the commits into 1 with rebase that would make things cleaner when I merge the pull request in.

Thanks!